backend/tests/integration/submissions/test_submissions_views.py

```diff
@@ -384,12 +385,81 @@ def test_new_literature_submit_no_merge(app, api_client, create_user, requests_m
             "document_type": ["article"],
             "inspire_categories": [{"term": "Other"}],
             "titles": [{"source": "submitter", "title": "Discovery of cool stuff"}],
+            "urls": [
+                {"value": "https://cern.ch/coolstuff.pdf"},
+                {"value": "https://cern.ch/other_stuff.pdf"},
+            ],
         },
         "form_data": {"references": "[1] Dude", "url": "https://cern.ch/coolstuff.pdf"},
```

Comment:
do we want the ``pdf_link`` to ``form_data``?

Reply:
Yes, we put "form_data' to workflow.extra_data['formdata'] and pdf_link should be there as without it workflows are complaining, but I never went deeper into it to see why.